app/controllers/items_controller.rb

```diff
@@ -165,6 +184,10 @@ def deny_return
                                                             date: Time.zone.now, active: false, unread: true)
     @declined_notification.save
     @item.destroy
+
+    # Comment out because an item gets deleted, when declining a return
+    # helpers.audit_deny_return(@item)
+
```

Comment:
I would remove this commented-out code as it might confuse other people working on the lending process.
```suggestion
```